### PR TITLE
Update Permission_Resource.postman_collection.json

### LINKED ISSUE #27244: Permission_Resource.postman_collection.json

### PR DESCRIPTION
**Pull request by rsh1k**
### Proposed Changes
Added the following:
* Postman test to check for sensitive data leakage.
* Test to check whether the generated IDs are unique or not. 
* Test to ensure the response includes a valid content type.

### Checklist
- [ *] Tests: All the tests mentioned above are added. 
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### CODE REVIEW

File: dotCMS/src/curl-test/Permission_Resource.postman_collection.json [hunk 1]
```diff
@@ -91,7 +103,34 @@
 									"});",
 									"pm.test(\"Icon should be 360\", function() {",
 									"    pm.expect(contentType.icon).to.eql(\"360\");",
-									"});"
+									"});",
+									"",
+-									"pm.test(\"Content type fields have expected structure\", function () {",
+-									"  const createdContentType = pm.response.json().entity[0];",
```
Comment (mbiuki) [inline dotCMS/src/curl-test/Permission_Resource.postman_collection.json:109]: These are removed?

Comment (bryanboza) [reply dotCMS/src/curl-test/Permission_Resource.postman_collection.json:109]: same thing, unable to import into postman, because have incorrect format

**Comment by mbiuki** [comment]
@rashik1144 please assign two reviewers from dev/qa team and also state what OWASP Top 10 tests you've added, thanks. Please apply this for all the tests your are adding to all collections moving forward, thanks

**Review by jdotcms — APPROVED** [review]

**Review by AndreyDotcms — APPROVED** [review]

**Comment by mbiuki** [comment]
@bryanboza - is this ready to merge?

**Review by nollymar — APPROVED** [review]

**Comment by mbiuki** [comment]
@nollymar - can we merge this?

**Pull request merged**